app/views/contests/summary.html.haml

```diff
@@ -43,7 +43,7 @@
   %h2 Uraian
   %table.table.table-condensed.table-striped.table-responsive
     %thead
-      = create_data_row(['No.', 'Soal', '-'] + (0..LongProblem::MAX_MARK).to_a + ['Rata-rata'],
+      = create_data_row(['No.', 'Soal', '-'] + (0..LongProblem.where(contest_id: params[:id]).maximum("max_score")).to_a + ['Rata-rata'],
```

Comment:
Can you extract this to a function in the `Contest` model?

e.g.
```ruby
def long_problem_max_score
  long_problems.maximum("max_score")
end
```

Reply:
okay